veg2hab/functionele_samenhang.py

```diff
@@ -225,9 +225,27 @@ def _remove_habtypen_due_to_minimum_oppervlak(
         keuzes = gdf.loc[gdf.ElmID == ElmID, "HabitatKeuze"].iloc[0]
         for index in indices:
             keuze_to_be_edited = keuzes[index]
+            assert not keuze_to_be_edited.habtype in [
+                "H0000",
+                "HXXXX",
+            ], "H0000 en HXXXX vlakken zouden uitgesloten moeten zijn van functionele samenhang"
+            assert keuze_to_be_edited.kwaliteit in [
+                Kwaliteit.GOED,
+                Kwaliteit.MATIG,
+            ], "Functionele Samenhangvlak kwaliteit moet GOED of MATIG zijn"
+
             keuze_to_be_edited.status = KeuzeStatus.MINIMUM_OPP_NIET_GEHAALD
-            keuze_to_be_edited.opmerking = f"Was {keuze_to_be_edited.habtype}, maar oppervlak was te klein. {keuze_to_be_edited.opmerking}"
+            keuze_to_be_edited.opmerking = (
+                "Was {} {}, maar oppervlak was te klein.{}".format(
+                    keuze_to_be_edited.habtype,
+                    "(" + keuze_to_be_edited.kwaliteit.as_letter() + ")",
```

Comment:
fstring? maakt ook weinig uit hoor :-)

Reply:
Dat wordt zo'n lange dan, zelfs als je hem (
   f"splits"
   f"op meerdere"
   f"regels"
)

Reply:
Mwah als ik er wat mee speel kan het best, maar dan verlies je wel het overzicht over hoe de string opgebouwd, ik vind het leesbaarder zoals het nu is

Reply:
Oh ik doelde eigenlijk op "(" + keuze_to_be_edited.kwaliteit.as_letter() + ")" als
```
f"({ keuze_to_be_edited.kwaliteit.as_letter() })"
```

Reply:
Strings op meerdere regels worden automatisch aan elkaar geknupt toch? Kan idd ook eventueel, maar dit is prima :-)